src/M6Web/Bundle/LogBridgeBundle/Matcher/Builder.php

```diff
@@ -77,88 +59,35 @@ class Builder implements BuilderInterface
     private $matcher;
 
     /**
+     * @var array
+     */
+    private $filters;
+
+    /**
+     * @var array
+     */
+    private $activeFilters;
+
+    /**
      * __construct
      *
      * @param StatusTypeManager $statusTypeManager Status type manager
-     * @param array             $resources         Resources
      * @param string            $environment       Environment name
+     * @param array             $filters           Filters
+     * @param array             $activeFilters     Active Filters
      */
-    public function __construct(StatusTypeManager $statusTypeManager, array $resources, $environment)
+    public function __construct(StatusTypeManager $statusTypeManager, array $filters, array $activeFilters, $environment)
     {
         $this->statusTypeManager = $statusTypeManager;
-        $this->resources         = $resources;
         $this->environment       = $environment;
-        $this->cacheResources    = [];
         $this->dispatcher        = null;
         $this->configParser      = null;
         $this->debug             = false;
         $this->cacheDir          = '';
         $this->matcherClassName  = '';
         $this->matcher           = null;
-    }
-
-    /**
-     * dispatch
-     * Dispatch event if EventDispatcher service exists
-     *
-     * @param string $eventName Event name
-     * @param Event  $event     Event class
-     *
-     * @internal param string $name Event name
-     */
-    protected function dispatch($eventName, $event)
-    {
-        if ($this->dispatcher) {
-            $this->dispatcher->dispatch($eventName, $event);
-        }
-    }
-
-    /**
-     * loadConfigResources
-     *
-     * @return array
-     */
-    protected function loadConfigResources()
-    {
-        $config = [];
-        $event  = new BuilderEvent($this, $config);
-
-        /*
-         * Dispatch event before Load and validate/compile resource
-         * From add or extends configuration
-         */
-        $this->dispatch(self::LOAD_RESOURCES_EVENT, $event);
-
-        $config = $event->getConfig();
-
-        foreach ($this->resources as $resource) {
-            $config = array_merge($config, $this->parse($resource));
-
-            $this->cacheResources[] = $resource;
-        }
-
-        $event->setConfig($config);
-        $this->dispatch(self::COMPILE_RESOURCES_EVENT, $event);
-
-        return $event->getConfig();
-    }
-
-    /**
-     * parse
-     *
-     * @param string $path File path
-     *
-     * @throws \Exception
-     *
-     * @return array
-     */
-    protected function parse($path)
-    {
-        if (!is_file($path) || !$content = file_get_contents($path)) {
-            throw new \Exception(sprintf('failed to open stream: No such file or is not readable "%s"', $path));
-        }
-
-        return Yaml::parse($content, true);
+        $this->filters           = $filters;
```

Comment:
Not very important, but these parameters should be between statusTypeManager and environnement